huxley/invoice_automation/src/model/conference.py

```diff
@@ -0,0 +1,11 @@
+from enum import Enum
+
+
+class Conference(Enum):
+    """
+    Enum representing different conferences
+    Used in Registration, and to determine which line items to use when creating invoices
+    """
+    BMUN71 = "BMUN 71"
```

Comment:
can you instead parse the current conference from the conference model associated with registrations? additionally, we don't support registration through FC so that isn't need. we want to avoid extra hard coding of information that can be parsed from models

Reply:
FC was needed for fc_handler, since inside QuickBooks itself we have a separate item. Will make BMUN change though

Reply:
Resolving: will remove FC once I put stuff in a separate repo